content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md

```diff
@@ -798,53 +798,100 @@ updates:
+
+Example:
+
+<table>
+    <thead>
+        <tr>
+            <th align="center">Constraint</th>
+            <th align="center">Current Version</th>
+            <th align="center">New Version</th>
+            <th>Strategy</th>
+            <th>New Constraint</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td rowspan="3" align="center"><code>^1.0.0</code></td>
+            <td rowspan="3" align="center"><code>1.0.0</code></td>
+            <td rowspan="3" align="center"><code>1.2.0</code></td>
+            <td align="center"><code>'widen'</code></td>
+            <td><code>^1.0.0</code></td>
+        </tr>
+        <tr>
+            <td align="center"><code>'increase'</code></td>
+            <td><code>^1.2.0</code></td>
+        </tr>
+        <tr>
+            <td align="center"><code>'increase-if-necessary'</code></td>
+            <td><code>^1.0.0</code></td>
+        </tr>
+      <tr>
+            <td rowspan="3" align="center"><code>^1.0.0</code></td>
+            <td rowspan="3" align="center"><code>1.0.0</code></td>
+            <td rowspan="3" align="center"><code>2.0.0</code></td>
+            <td align="center"><code>'widen'</code></td>
+            <td><code>&gt;=1.0.0 &lt;3.0.0</code></td>
+        </tr>
+        <tr>
+            <td align="center"><code>'increase'</code></td>
+            <td><code>^2.0.0</code></td>
+        </tr>
+        <tr>
+            <td align="center"><code>'increase-if-necessary'</code></td>
+            <td><code>^2.0.0</code></td>
+        </tr>
+    </tbody>
+</table>
 
 Use the `versioning-strategy` option to change this behavior for supported package managers.
 
 {% data reusables.dependabot.option-affects-security-updates %}
 
-Available update strategies
-
-| Option | Supported by | Action |
-|--------|--------------|--------|
-| `lockfile-only` | `bundler`, `cargo`, `composer`, `mix`, `npm`, `pip` | Only create pull requests to update lockfiles. Ignore any new versions that would require package manifest changes. |
-| `auto` | `bundler`, `cargo`, `composer`, `mix`, `npm`, `pip` | Follow the default strategy described above.|
-| `widen`| `composer`, `npm` | Relax the version requirement to include both the new and old version, when possible. |
-| `increase`| `bundler`, `composer`, `npm`{% ifversion dependabot-increase-version-pip-support %}, `pip`{% endif %}  | Always increase the version requirement to match the new version. |
-| `increase-if-necessary` | `bundler`, `composer`, `npm`{% ifversion dependabot-increase-version-pip-support %}, `pip`{% endif %} | Increase the version requirement only when required by the new version. |
+Available update strategies:
+
+| Ecosystem | Supported _versioning-strategies_ | Default strategy | Additional notes |
+|-----------|-----------------------------------|------------------|------------------|
+| `bundler` | `auto`, `increase`, `increase-if-necessary`, `lockfile-only` | `auto` |
```

Comment:
We don't seem to really need the "Additional notes" column as it seems to be a repeat of the `Default strategy` column, unless I misunderstood its purpose. Shall we remove it?